### PR TITLE
Add simplehtml chart for simple static sites on SILTA

### PR DESCRIPTION
**Pull request by gatis**
Story: SLT-171 
This feature adds simple html chart for hosting simple static sites on SILTA 

Testing: deploy project with  `helm upgrade --install $RELEASE_NAME simplehtml --repo https://wunderio.github.io/charts/ .. `

### CODE REVIEW

**Comment by floretan** [comment]
I would pick a more generic name than "simplehtml". The same chart could be used for a node.js node service serving a react application as well as the data for it over graphql, it's just a matter of what we put in the main container image. How about "simple", since the chart is pretty much for applications running on a single, stateless container?

**Comment by gatis** [comment]
I agree, use-cases could be broader than hosting plain html.

File: docs/index.yaml [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
     version: 0.1.0
-generated: 2018-11-23T20:41:26.97409662+01:00
+  simple:
+  - created: 2018-11-27T09:26:53.710570657+02:00
+    digest: ba25ce56484b9b2f4602290aac5e58b5a97928c3758606a2d7da2110d8d70e90
+    name: simple
+    urls:
+    - https://wunderio.github.io/charts/simple-0.1.2.tgz
+    version: 0.1.2
+  simplehtml:
```
Comment (floretan) [inline docs/index.yaml]: There is still an outdated reference to simplehtml.

Comment (gatis) [reply docs/index.yaml]: I did not delete .tgz so apparently build left int in index.
Should be fixed now.

**Pull request merged**